kimchi/src/tests/lazy_mode.rs

```diff
@@ -0,0 +1,85 @@
+use super::framework::TestFramework;
+use crate::circuits::{
+    gate::CircuitGate,
+    polynomial::COLUMNS,
+    polynomials::{generic::GenericGateSpec, xor},
+    wires::Wire,
+};
+use ark_ff::Zero;
+use itertools::iterate;
+use mina_curves::pasta::{Fp, Vesta, VestaParameters};
+use mina_poseidon::{
+    constants::PlonkSpongeConstantsKimchi,
+    sponge::{DefaultFqSponge, DefaultFrSponge},
+};
+use rand::Rng;
+use std::array;
```

Comment:
Can you replace by `core::array`? We try to use `core` as much as possible instead of `std`.

Reply:
Oh that's true